api/src/main/java/com/spectralogic/tpfr/api/ServerServiceFactoryImpl.java

```diff
@@ -0,0 +1,73 @@
+package com.spectralogic.tpfr.api;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.logging.HttpLoggingInterceptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import retrofit2.Retrofit;
+import retrofit2.converter.simplexml.SimpleXmlConverterFactory;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import static okhttp3.OkHttpClient.Builder;
+
+public class ServerServiceFactoryImpl implements ServerServiceFactory {
+
+    private final Logger LOG = LoggerFactory.getLogger(ServerServiceFactoryImpl.class);
+    private ServerService serverService;
+
+    @Override
+    public Optional<ServerService> getServerService() {
+        return Optional.ofNullable(serverService);
```

Comment:
I'm confused, why would this ever return a nullable value?  I don't see any exceptions possible when creating the service.